oximeter/db/src/lib.rs

```diff
@@ -117,136 +125,6 @@ pub enum Error {
     NonSequentialSchemaVersions,
 }
 
-/// A timeseries name.
```

Comment:
Almost all of this negative diff was just moved into `oximeter/src/types.rs` and re-exported here.